modules/javafx.graphics/src/main/java/javafx/scene/CssStyleHelper.java

```diff
@@ -131,7 +131,10 @@ static CssStyleHelper createStyleHelper(final Node node) {
                 node.styleHelper.cacheContainer.fontSizeCache.clear();
             }
             node.styleHelper.cacheContainer.forceSlowpath = true;
-            node.styleHelper.triggerStates.addAll(triggerStates[0]);
+
+            if (triggerStates[0] != null) {
```

Comment:
I see that a null check was added.
Are we sure we added it to *every* possible place?

(it's not trivial to verify that with Eclipse without massive changes)

Reply:
The underlying collection `BitSet` is only used by classes dealing with styles and pseudo class states. I checked the classes involved, and only this one needed changing.  The check was added because `BitSet` erroneously allowed adding `null` before (which resulted in no change), but that breaks the `Set` contract.

In any case, adding `null` is an error, we should never be lenient when it comes to `null`s.